src/helm/benchmark/run_specs/experimental_run_specs.py

```diff
@@ -192,3 +192,34 @@ def get_czech_bank_qa_spec(config_name: str = "berka_queries_1024_2024_12_18") -
         annotators=[AnnotatorSpec("helm.benchmark.annotation.czech_bank_qa_annotator.CzechBankQAAnnotator")],
         groups=["czech_bank_qa"],
     )
+
+
+@run_spec_function("infinite_bench_sum")
+def get_infinite_bench_sum_spec(word_lower_bound: float = 0.0, word_upper_bound: float = 100e6) -> RunSpec:
```

Comment:
`1e8` instead of `100e6` (also isn't 1e7 enough?)

Reply:
yes 1e7 is enough; re 1e8 vs 100e6, was meant to make it easier to understand (100e6=100M, 30e3=30k, etc), but am fine with both. 
Currently: changed to 1e7 in the latest change